modules/survey_accounts/jsx/surveyAccountsIndex.js

```diff
@@ -0,0 +1,136 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class SurveyAccountsIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
+   * for easy access by columnFormatter.
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({data}))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
```

Comment:
Should we remove console.error

Reply:
I think it's been standard practice to have the error be logged. I wasn't there for that decision though and have simply been putting it there for legacy reasons. I'm really neither here not there on this one. Don't mind if it stays, don't mind if it goes, as long as we stay consistent.

Reply:
@HenriRabalais Our .eslintrc.json warns when finding a console in the code. I'm either side as well but it would be nice to remove as many warnings from populating and when we run `npm run compile` in the terminal.